soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -0,0 +1,1944 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CHIP_CHIPREGS_H
+#define CHIP_CHIPREGS_H
+
+#include <sys/util.h>
+
+#define EC_Register_Base_Address DT_REG_ADDR_BY_IDX(DT_NODELABEL(baseaddr), 0)
+
+#ifndef FALSE
+#define FALSE   0
+#endif
+
+/* TRUE can be defined as !FALSE but defining
+ * it as 1 allows it to fit into a bitfield.
+ */
+#ifndef TRUE
+#define TRUE    1
+#endif
+
+#ifdef _ASMLANGUAGE
+#define ECReg(x)        x
+#else
+
+/*
+ * Macros for hardware registers access.
+ */
+#define ECReg(x)		(*((volatile unsigned char *)(x)))
```

Comment:
all caps for macro identifiers please

probably should just define in terms of sys_write32, sys_write16, etc

also it seems every usage is `ECReg(EC_Register_Base_Address + <some address>)` why not incorporate the base address into the definition instead of repeating ad nauseam?

Reply:
Hi,

thank for reviewing,
Macro has been fixed to be capital. 
The current ECREG define is flexiable for our patform, so we keep it.
The base address or the register function base address may change with different chip, so we prefer using this way for our chip.